libc/test/src/math/smoke/CanonicalizeTest.h

```diff
@@ -45,6 +48,104 @@ class CanonicalizeTest : public LIBC_NAMESPACE::testing::Test {
     EXPECT_EQ(cx, -aNaN);
   }
 
+  void testX64_80SpecialNumbers(CanonicalizeFunc f) {
```

Comment:
Put the whole test under: `if constexpr( LIBC_NAMESPACE::fputil::get_fp_type() == ...) { ... }`

Reply:
done.